src/App.js

```diff
@@ -943,14 +938,14 @@ export class App extends React.Component<AppProps, AppState> {
     handleToggleAllowedCommand = (event: Event, commandName: CommandName) => {
```

Comment:
The `handleToggleAllowedCommand()` method isn't used and can be removed.

Reply:
Fixed.